tests/testthat/test-resample.R

```diff
@@ -178,8 +176,20 @@ test_that("can use `fit_resamples()` with a workflow - postprocessor (requires t
   seed <- generate_seeds(TRUE, 1)[[1]]
   old_kind <- RNGkind()[[1]]
   assign(".Random.seed", seed, envir = globalenv())
+  withr::defer(RNGkind(kind = old_kind))
```

Comment:
An RNG state oddity—without it, tests elsewhere in the test suite begin failing with RNG changes.